### ISSUE TITLE
关于类似自定义抽屉式菜单或者底部菜单的多个顶层导航的功能无法实现

### ISSUE THREAD

**Issue by AnthonyCoder** [vitaviva/fragivity#52]
从Navigation过来的,项目迁移,遇到一个问题,关于自定义的抽屉菜单无法配合使用多顶层导航实现Fragment之间的管理
见:[官方的抽屉式多顶层导航实现示例代码](https://developer.android.com/guide/navigation/navigation-ui#add_a_navigation_drawer)

起因是这样,因为项目使用了自定义布局的抽屉式写法类似([利用DrawerLayout实现自定义左右抽屉布局](https://juejin.cn/post/6844903682111193096))

但是这样在放弃使用官方样式的同时也抛弃了NavigationView的使用和menu相关的配置
所以没办法通过Navigation去实现同时多个顶层导航,
然后看了下fragivity的示例代码,
发现在Activity中去通过loadRoot()方法设置好像也只可能有一个顶层导航,
所以希望大佬看看有没有实现的可能,
如果有的话,可以麻烦在代码中加上相关示例吗?
因为我认为这是一个强需求

**Comment by AnthonyCoder** [comment]
另外,这个开源库很棒,作者大大加了不少常用且通用的扩展函数,从使用上来说相比Navigation,代码量精简了不少!

**Comment by qdsfdhvh** [comment]
多返回栈吗，[新版navigation](https://www.youtube.com/watch?v=Covu3fPA1nQ)是支持了，前段时间尝试适配了下(feature/multi-back分支)，但是还没实现生命周期走到onStart/onStop，不太好弄。
不是多返回栈的话，建议用viewpager2。

**Comment by AnthonyCoder** [comment]
是的,但是官方的example都是基于他自己的组件,比如抽屉菜单需要NavigationView和menu文件绑定,底部菜单BottomNavigation和menu绑定.
那如果我不是用官方的UI组件,比如我想要自定义一些组件就没办法实现多返回栈了.

**Comment by qdsfdhvh** [comment]
可以参照官方来适配自定义组件，虽然实现的代码不会很好看但是应该能实现；

后面会尝试增加loadMultiRoot()，就是不怎么好实现，加上最近工作上也比较忙，可能会要些时间。
